system.c

```diff
@@ -379,8 +379,11 @@ uint8_t system_execute_line(char *line)
             case 'R':
             {
               const uint8_t result = ad5121_read_pot((enum AD5121_ID)dev_id);
+	      printPgmString(PSTR("[POT<"));
```

Comment:
To rephrase, don't mix tabs and spaces. There ought to be a way to configure emacs to turn tabs into spaces automatically. It's easy in vim... :-)